webhook/webhook.yaml

```diff
@@ -14,14 +14,14 @@ components:
     WebhookMessage:
       type: object
       required:
+        - action
         - projectId
         - provider
         - groupRef
         - groupName
         - installationId
         - objectName
         - resultInfo
-        - result
```

Comment:
This will turn `Results` into a `*[]openapi.ResultEntry`, so builders using this will have to update their code if they are using it